### PR TITLE
fix #280

### LINKED ISSUE #280: test fails because language is not english.
It seems that the test is based on english, the element selecter is based on the language-spec selecter, so lot's of test fails because of unable to find the element.
for example `$("*[aria-label='Toggle mute audio']")` will find nothing, but `$("*[aria-label='静音 / 取消静音']")` will find the correct element on my computer.

### PR DESCRIPTION
**Pull request by vipcxj**
fix #280 
Force chrome and firefox web driver use English instead of system language.
The code for chrome has test on my computer. The chrome version is 83.0.4103.61. The os system is win 10.

### CODE REVIEW

File: src/test/java/org/jitsi/meet/test/web/WebParticipantFactory.java [hunk 1]
```diff
@@ -203,6 +206,9 @@ else if (participantType == ParticipantType.edge)
             logPrefs.enable(LogType.BROWSER, Level.ALL);
 
             final ChromeOptions ops = new ChromeOptions();
+            // Force chrome to use English instead of system language.
+            // At least work at version 83.0.4103.61 on win 10
+            ops.addArguments("--lang=en");
```
Comment (damencho) [inline src/test/java/org/jitsi/meet/test/web/WebParticipantFactory.java:211]: I think these arguments need to be without `--`

Comment (vipcxj) [reply src/test/java/org/jitsi/meet/test/web/WebParticipantFactory.java:211]: It should with --

Comment (vipcxj) [reply src/test/java/org/jitsi/meet/test/web/WebParticipantFactory.java:211]: There is another solution.
```java
            Map<String, String> lops = new HashMap<>();
            lops.put("intl.accept_languages", "en,en_US");
            ops.setExperimentalOption("prefs", lops);
```
It work on my computer, too.
But `addArguments` is short, so I choose that.

Comment (damencho) [reply src/test/java/org/jitsi/meet/test/web/WebParticipantFactory.java:211]: Look at the rest of the arguments like `ops.addArguments("allow-insecure-localhost");`
On the command line that is `--allow-insecure-localhost`.

Comment (vipcxj) [reply src/test/java/org/jitsi/meet/test/web/WebParticipantFactory.java:211]: with `--` it just work. without `--` I can't promise it will still work.

Comment (damencho) [reply src/test/java/org/jitsi/meet/test/web/WebParticipantFactory.java:211]: You had tested it with torture and -- ? Or you just had tested it -- command line?

Comment (damencho) [reply src/test/java/org/jitsi/meet/test/web/WebParticipantFactory.java:211]: Or you are right, it does not matter https://www.selenium.dev/selenium/docs/api/java/org/openqa/selenium/chrome/ChromeOptions.html#addArguments-java.util.List-

Comment (damencho) [reply src/test/java/org/jitsi/meet/test/web/WebParticipantFactory.java:211]: As you can see it the docs the examples are without --

Comment (vipcxj) [reply src/test/java/org/jitsi/meet/test/web/WebParticipantFactory.java]: I have test without `--` on windows, it also works, so I removed the `--` to keep the format.

**Comment by vipcxj** [comment]
I found some other bugs when using this test framework on windows. Such as send text to the input sometimes crash because the input is invalid. I think it perhaps because the react update the component when the input is focused.

**Pull request merged**